packages/@tinacms/cli/src/logger/index.ts

```diff
@@ -15,3 +13,208 @@ log4js.configure({
 })
 // set initial level to info
 logger.level = 'info'
+
+// Used from https://github.com/natemoo-re/clack/blob/main/packages/prompts/src/index.ts
+function ansiRegex() {
+  const pattern = [
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-nq-uy=><~]))',
+  ].join('|')
+
+  return new RegExp(pattern, 'g')
+}
+const bar = '│'
+const strip = (str: string) => str.replace(ansiRegex(), '')
+export const note = (message = '', title = '') => {
+  const lines = `\n${message}\n`.split('\n')
+  const len =
+    lines.reduce((sum, ln) => {
+      ln = strip(ln)
+      return ln.length > sum ? ln.length : sum
+    }, 0) + 2
+  const msg = lines
+    .map(
+      (ln) =>
+        `${chalk.gray(bar)}  ${chalk.white(ln)}${' '.repeat(
+          len - strip(ln).length
+        )}${chalk.gray(bar)}`
+    )
+    .join('\n')
+  process.stdout.write(
+    `${chalk.gray(bar)}\n${chalk.green('○')}  ${chalk.reset(
+      title
+    )} ${chalk.gray(
+      '─'.repeat(len - title.length - 1) + '╮'
+    )}\n${msg}\n${chalk.gray('├' + '─'.repeat(len + 2) + '╯')}\n`
+  )
+}
+export function logCreateAppResult(
+  pkgManager: string,
+  result: { outDir: string; docs: string[] },
+  ranInstall: boolean
+) {
+  const outString = []
+
+  // outString.push(
+  //   `🦄 ${chalk.bgMagenta(' Success! ')} ${chalk.cyan(
+  //     `Project created in`
+  //   )} ${chalk.bold(chalk.magenta('.tina'))} ${chalk.cyan(`directory`)}`
+  // )
+  // outString.push(``)
+
+  outString.push(`🦙 ${chalk.cyan(`Tina Config:`)}`)
+  outString.push(
+    `   API url:        ${chalk.cyan('http://localhost:4001/graphql')}`
+  )
+  outString.push(
+    `   API playground: ${chalk.cyan('http://localhost:4001/altair')}`
+  )
+  outString.push(
+    `   Tina CMS Admin: ${chalk.cyan('<your-dev-server>/admin/index.html')}`
+  )
+  outString.push(``)
+
+  outString.push(`🤍 ${chalk.cyan('Auto-generated files...')}`)
+  outString.push(
+    `   GraphQL Client: ${chalk.cyan('.tina/__generated__/client')}`
+  )
+  outString.push(
+    `   Typescript Types: ${chalk.cyan('.tina/__generated__/types')}`
+  )
+  outString.push(``)
+
+  outString.push(`👀 ${chalk.cyan('Useful links')}`)
+  outString.push(
+    `   Custom queries: ${chalk.cyan('https://tina.io/querying/')}`
+  )
+  outString.push(
+    `   Visual editing: ${chalk.cyan('https://tina.io/visual-editing/')}`
+  )
+  outString.push(``)
+
+  return outString.join('\n')
+}
+
+export const log = (message = '') => {
```

Comment:
Should we be using our Logger module here?

Reply:
I'm hoping to combine these in the near future. It's all in the logger module for now regardless.